base-server/index.js

```diff
@@ -355,6 +369,45 @@ export class BaseServer {
     this.listenNotes = {}
     bindBackendProxy(this)
 
+    let end = (meta, queue, queueKey, ...args) => {
```

Comment:
```suggestion
    function end (meta, queue, queueKey, ...args) {
```

A small style fix to keep code style consistent